src/daos/color_output.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
```

Comment:
These loose scripts could be grouped into directories: daos-related scripts, radosgw stuff, utils

Reply:
I'll do that after the squash